elastic/shared/runners/remote_cluster.py

```diff
@@ -1,143 +1,179 @@
+import asyncio
+import copy
 import math
 import time
 
 from elasticsearch import ElasticsearchException
 from esrally.driver.runner import Runner
 
 """
-Runners for benchmarking multi-clusters. Once these runners are stable, we can move them to Rally.
```

Comment:
Just curious, as this is no longer in the comment. Do you feel like they're not suitable for Rally?

> Once these runners are stable, we can move them to Rally.

Reply:
IMHO, these runners are specific enough to the implementation of the challenge that they don't make sense to implement as core runners. Additionally, we do actually indicate that [_all_ multi-clusters runners should be custom](https://esrally.readthedocs.io/en/latest/command_line_reference.html#id1):
> <h1> target-hosts </h1>
> Rally can also create client connections for multiple Elasticsearch clusters. This is only useful if you want to create [custom runners](https://esrally.readthedocs.io/en/latest/adding_tracks.html#adding-tracks-custom-runners) that execute operations against multiple clusters, for example for [cross cluster search](https://www.elastic.co/guide/en/elasticsearch/reference/current/modules-cross-cluster-search.html) or cross cluster replication.